### PR TITLE
#CNV-30307: new rn for localnet topology

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.15 RN

Issue: [CNV-30307](https://issues.redhat.com//browse/CNV-30307)

Link to docs preview: https://71885--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/release_notes/virt-4-15-release-notes#virt-4-15-networking

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: virt/release_notes/virt-4-15-release-notes.adoc [hunk 1]
```diff
@@ -96,6 +96,8 @@ This release adds new features and enhancements related to the following compone
 
 //CNV-30296: Running DPDK workloads on SR-IOV is now GA. The TP note from 4.13 should have been part of 4.14 as well.
 
+//CNV-30307
+* You can now xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-creating-nad-localnet-console_virt-connecting-vm-to-ovn-secondary-network[create a network attachment definition (NAD) for localnet topology] by using the {product-title} web console
```
Comment (skopacz1) [inline virt/release_notes/virt-4-15-release-notes.adoc:100]: Missing a period here:
```suggestion
* You can now xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-creating-nad-localnet-console_virt-connecting-vm-to-ovn-secondary-network[create a network attachment definition (NAD) for localnet topology] by using the {product-title} web console.
```

**Review by skopacz1 — COMMENTED** [review]
One nit, otherwise LGTM!

**Comment by sabrinajess** [comment]
@gouyang Please provide QE review.

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**